src/app/shared/components/video-player/video-player.component.ts

```diff
@@ -43,6 +70,12 @@ import {SanitizeUrlPipe} from "../../pipes/sanitize-url.pipe";
   `
 })
 export class VideoPlayerComponent {
-  src = input.required<string>();
+  readonly videoSrc = input.required<string>();
+  readonly placeholderSrc = input.required<string>();
   title = input<string>('');
+  isLoaded = false;
+
+  loadIframe() {
+    this.isLoaded = true;
+  }
```

Comment:
Könnte man auch zu einem Signal machen. Ich pack mittlerweile eigentlich alles in Signals, weil dann einfach weniger Leseanfragen getriggert werden. Hier eine Diskussion dazu: [Link](https://www.reddit.com/r/Angular2/comments/1cgpeq5/comment/l1ygce1/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) 

```suggestion
  isLoaded = signal(false);

  loadIframe() {
    this.isLoaded.set(true);
  }
```


Zeile 16 müsstest du dann auch noch anpassen.